payjoin-cli/Cargo.toml

```diff
@@ -19,8 +19,10 @@ path = "src/main.rs"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 [features]
 native-certs = ["ureq/native-certs"]
-danger-local-https = ["rcgen", "rustls", "hyper-rustls"]
-v2 = ["payjoin/v2"]
+danger-local-https = ["rcgen", "rustls", "hyper-rustls", "payjoin-defaults/danger-local-https"]
+v2 = ["payjoin/v2", "payjoin-defaults/v2"]
+
+default = ["danger-local-https", "v2"]
```

Comment:
`danger-local-https` is a test-only feature. If it is used by default then the CLI tool won't check https certs. That's dangerous! We can't ship a production build with this feature set. Where did the idea to set this default come from?

Reply:
aye sorry that's a leftover from some manual local testing